### ISSUE TITLE
Not working in Safari

### ISSUE THREAD

**Issue by KyruCabading** [closeio/mic-recorder-to-mp3#7]
Not Supported for safari, I'm getting this error. 
`ReferenceError: Can't find variable: AudioContext`

Not familiar with working on this code but I'd like to try and help! I'll make a fork and try to fix it there.

References I will use:
https://github.com/amaneureka/T-Rex/issues/5
https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&ved=2ahUKEwiFgJPCsMfhAhUKZt4KHaAkDesQFjAAegQIBhAB&url=https%3A%2F%2Fstackoverflow.com%2Fquestions%2F29373563%2Faudiocontext-on-safari&usg=AOvVaw3kEDo36YZzRZyMotS4zQRj

**Comment by eltongonc** [comment]
@KyruCabading  I've noticed that setting the window.AudioContext helped me fix a part of this issue on the newer safari (IOS 12). 
```js
window.AudioContext = window.AudioContext || window.webkitAudioContext;
```

**Comment by orizens** [comment]
@KyruCabading 
review support for AudioContext
https://caniuse.com/#feat=mdn-api_audiocontext

**Comment by LovaszNorbert** [comment]
there is Still a issue in Safari , after we make the change with webkitAudioContext , the recording is so messed Up , distorted, does someone have any idea why?

**Comment by JijeshP** [comment]
@LovaszNorbert facing the same issue, did u find any solution for this?

**Comment by brunoqs** [comment]
facing the same issue too

**Comment by erinwilder** [comment]
I am also getting the problem where the recording is distorted in safari, but only on mobile.

**Comment by andraderaul** [comment]
I have the same issue, any solution?

**Comment by buzinas** [comment]
The original issue was fixed by #8. The webkit issue, you should probably file a bug in Safari's repo.
